### DISCUSSION TITLE
Blocks and items from a mod

### DISCUSSION THREAD

**Discussion by Ja90n** [Minestom/Minestom#1982] [Help]
Hello, is there any way to use items / blocks from a mod? To add support for another namespace beside Minecraft's?
Thanks!

**Comment by Flopgop** [comment]
Yes there is a way, iirc the correct way to do this is to modify the `blocks.json` and `items.json` that are generated by datagen to include your modded blocks and items. You can rip the default jsons from [Maven Central](https://repo1.maven.org/maven2/net/minestom/data/) and put your modified `blocks.json` and `items.json` in your resources folder.
